lib/controller.js

```diff
@@ -92,7 +95,7 @@ Tessel.list = function(opts) {
 
 Tessel.get = function(opts) {
   return new Promise(function(resolve, reject) {
-    logs.info('Looking for your Tessel...');
+    logs.info('Searching nearby LAN (netmask) and USB for Tessels...');
```

Comment:
@rwaldron didn't notice this until I added the current experimental `t2 select` command. But in this case this could be a switch around `opts.lan`:
```
if(opts.lan) {
  logs.info('Searching nearby LAN (netmask) for Tessels...');
} else if(opts.usb) {
  logs.info('Searching USB for your Tessel...');
} else {
  logs.info('Searching nearby LAN (netmask) and USB for Tessels...');
} 
```
**Question**: Because I do not own a single Tessel 2 .. ... . I don't know whether `var seeker = new Seeker ..` also discovers USB Tessels (I assume) ... but does it differ between `--lan`and `--usb` or is this done by filtering later ?